src/main/java/seedu/duke/ExpenditureList.java

```diff
@@ -13,6 +13,7 @@ public ExpenditureList() {
     }
 
     public static void listExpensesByMonth(String monthYear) {
+        assert monthYear.length() == 7;
```

Comment:
magic number?

Reply:
Because from Jan to July there are 7 months

Reply:
Because the string with "mm.yyyy" is 7 string long.

Reply:
Put constant value for better coding standards

Reply:
> Because from Jan to July there are 7 months

👎

Reply:
Okie :((((((((((((((9